### PR TITLE
Wrpd enhancement 169

### PR DESCRIPTION
**Pull request by balasai45**
## Description

Created a generic Swagger Generate functionality which fetches data from database and populated the swagger json

## Related Issues

#169 It resolves this issue

## Checklist

- [ ] I have performed a thorough self-review of my code.
- [ ] My code follows the project's style guidelines and best practices.
- [ ] I have added or updated relevant comments in my code.
- [ ] I have resolved any merge conflicts of my branch.

## Screenshots (if applicable)
image
image

## Reviewers

@techoneel

### CODE REVIEW

File: package/WrappidApp.ts [hunk 1]
```diff
@@ -66,7 +68,7 @@ export default class WrappidApp {
     },
     swagger: {},
     config: {},
-    package: {}
+    package: {},
```
Comment (techoneel) [inline package/WrappidApp.ts:71]: Additional comma

File: package/swagger/swagger-generate.ts [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+
+export const generateSwaggerJson = async (swaggerJson: GenericObject) => {
+  const data = await databaseActions.findAll("application", "Routes", {
+    where: {
+      source: "server-side",
+      _status: "published",
+
+    }
+  });
+  const models = ApplicationContext.getContext(constant.registry.MODELS__REGISTRY);
```
Comment (techoneel) [inline package/swagger/swagger-generate.ts:103]: Type declaration missing

**Review by techoneel — CHANGES_REQUESTED** [review]
Please work on the requested changes and commit again.

**Comment by balasai45** [comment]
Committed fixes

**Comment by wrappidcare** [comment]
Merging since stale over 24hrs.

**Pull request merged**